modules/misc/news.nix

```diff
@@ -1033,6 +1033,12 @@ in
           defaults to 'pkgs.openssh'.
         '';
       }
+      {
+        time = "2023-05-18T21:03:30+00:00";
+        message = ''
+          A new module is available: 'programs.shell-directory'.
```

Comment:
Shouldn't this be `programs.script-directory`?

Reply:
Thanks, fixed